voila/app.py

```diff
@@ -566,7 +566,7 @@ def listen(self):
         success = False
         for port in self.random_ports(self.port, self.port_retries+1):
             try:
-                self.app.listen(port)
+                self.app.listen(port,address=self.ip)
```

Comment:
Thank you very much for your contribution! A minor code format problem:

```suggestion
                self.app.listen(port, address=self.ip)
```